src/docs/product/projects/project-details/index.mdx

```diff
@@ -4,47 +4,140 @@ sidebar_order: 1
+
+<div style="position: relative; padding-bottom: calc(54.296875% + 41px); height: 0; width: 100%">
+  <iframe
+    src="https://demo.arcade.software/XU3D5nw7kEDBO9KwmwuR?embed"
+    frameborder="0"
+    loading="lazy"
+    webkitallowfullscreen
+    mozallowfullscreen
+    allowfullscreen
+    style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;color-scheme: light;"
+    title="Project Details -> Graphs"
+  ></iframe>
+</div>
 
-The two graphs below the cards provide insight into:
+By default, the graphs show crash free sessions and Apdex, but you can toggle the **Display** to set the view to any of the following:
 
 - Crash free [sessions](/product/releases/health/#session)
+- Crash free users
 - [Apdex](/product/performance/metrics/#apdex)
 - [Failure rate](/product/performance/metrics/#failure-rate)
 - [Transactions per minute](/product/performance/metrics/#throughput-total-tpm-tps)
-- Daily errors
-- Daily transactions
+- Number of errors
+- Number of sessions
+- Number of transactions
+
+You can also use this page to set up [alerting](/product/alerts-notifications/) or [triage an issue](#triage).
+
+## Create an Alert
 
-Toggle the **Display** to change the view. You can click on the graph legend to show or hide the overlay of releases and previous period.
+You can use the **Project Details** page to access Sentry's alerting feature.
 
-## Alerting
+Click the **Create Alert** button at the top right to set up either an [Issue Alert](/product/alerts-notifications/issue-alerts/) or a [Metric Alert](/product/alerts-notifications/metric-alerts/).
 
-You can use the Project Details page to access Sentry's alerting feature. Click the **Create Alert** button at the top right to set up either an [Issue Alert](/product/alerts-notifications/issue-alerts/) or a [Metric Alert](/product/alerts-notifications/metric-alerts/). In the **Latest Alerts** section of the sidebar you can see the three latest triggered alerts. Click on one of them to go to Alert Details page. The selected time range does not affect the displayed alerts.
+<div style="position: relative; padding-bottom: calc(61.72148355493352% + 41px); height: 0; width: 100%">
+  <iframe
+    src="https://demo.arcade.software/ZEOZDNURwYhmkqdQ76ge?embed"
+    frameborder="0"
+    loading="lazy"
+    webkitallowfullscreen
+    mozallowfullscreen
+    allowfullscreen
+    style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;color-scheme: light;"
+    title="Project Details -> Create Alert"
+  ></iframe>
+</div>
 
-## Releases
+## View Latest Alerts
 
-You can use the Project Details page to assess how often you release new versions of your application:
+The **Latest Alerts** section in the right sidebar shows the 3 latest triggered alerts. Click on an alert to go to the **Alert Details** page. The selected time range does not affect the displayed alerts.
+
+<div style="position: relative; padding-bottom: calc(61.72148355493352% + 41px); height: 0; width: 100%">
```

Comment:
This one looks a little blurry for some reason. I also couldn't tell what step 3 was for.